app/assets/javascripts/listings/components/listing/features-section.html.slim

```diff
@@ -47,27 +47,27 @@ accordion-heading
             svg
               use xlink:href="#i-arrow-down"
 
-      .stats.scrollable-table-container ng-show="$ctrl.parent.toggleStates[$ctrl.parent.listing.Id][unitGroup.unitType]"
+      .stats.scrollable-table-container ng-show="$ctrl.parent.toggleStates[$ctrl.parent.listing.id][unitGroup.unitType]"
         table role="grid"
           thead
             tr
               th scope="col" Unit
               th scope="col" Area
               th scope="col" Baths
               th scope="col" Floor
-              th scope="col" Accessibility
+              th scope="col" accessibility
```

Comment:
@jaredcwhite should we keep this as "Accessibility" to match the casing of the other column headers here?